integration_tests/tests/pages_are_associated_to_page_definition.sql

```diff
@@ -0,0 +1,4 @@
+select
+  *
+from {{ ref('test_pages') }}
+where page_id IS NULL
```

Comment:
Getting linter(s) sorted out in this project would help with trivial issues like missing newlines. dbt recommends sqlfluff and/or sqlfmt though I found the experience to be lack luster.